sdk/metric/pipeline.go

```diff
@@ -134,3 +138,111 @@ func (p *pipeline) produce(ctx context.Context) (metricdata.ResourceMetrics, err
 		ScopeMetrics: sm,
 	}, nil
 }
+
+// pipelineRegistry manages creating pipelines, and aggregators.  The meters can
+// retrieve new aggregators from a registry.
```

Comment:
```suggestion
// pipelineRegistry manages creating pipelines, and aggregators.
// The meters can retrieve new aggregators from a registry.
```

Reply:
I took the other rewording of this comment.